### ISSUE TITLE
Mode selection window is a trap. (Problem is also related to language selection window.)

### ISSUE THREAD

**Issue by Realchacal** [monero-project/monero-gui#2321]
(I made this an own issue, since it can be handled separately from other issues.)
![image](https://user-images.githubusercontent.com/46682965/61927179-10892000-af74-11e9-96e1-e0018d5d2bbd.png)

Problems with that:

- The mode selection window is a trap: You are forced to select something but you can't cancel/ go back.
- It doesn't even tell you the current mode. That causes anxiety, because users are afraid they might change something unintentionally.
- The mode selection window doesn't allow us to preselect a recommended mode during initial setup. But it would make much sense to preselect a privacy-preserving solution for the user.
- During initial setup the user gets presented a huge language select window. When he's done with that, he clicks continue. After that he gets presented another window that -again- contains an option "change language". User thinks: "??? - Didn't I just change the language!?"
- That "change language" button also makes no sense during regular wallet use. Why should just the mode selection window have an extra "change language" button, while no other window has it?

Exemplary solution for the mode selection window (with wording from #2320):
![image](https://user-images.githubusercontent.com/46682965/61932157-edb23800-af82-11e9-99bd-7fe6e1e32b4e.png)

...with some small changes this gives us a consistent initial setup with simple "<Back" and "Next>" buttons that everybody is used to:
![image](https://user-images.githubusercontent.com/46682965/61932255-2a7e2f00-af83-11e9-917d-563828687d0c.png)

**Comment by GBKS** [comment]
Yeah, the "Back" button labels were not implemented as [originally designed](https://www.dropbox.com/s/wec41pejuevlpqd/monero-onboarding-wallet-mode-gbks-190203.png?dl=0). Should always just  read "Back" (not use the previous screens name) and the buttons should be light grey so they are less prominent visually compared to the "Next" button. There was some confusion around this I saw in users during this recent [user testing session](https://paper.dropbox.com/doc/Monero-GUI-user-testing--Ahq0IskVq_SDsjTKEwvlQqAcAg-YAWmy01OJa5vkmdQvt6wg). The list of modes should also have a "selected" state (like they do in the settings page) with a "Next" button to confirm the choice. Definitely some small interaction tweaks that would improve usability.

I don't have a super strong opinion on whether the language picker is a dropdown or an independent screen.

**Comment by Realchacal** [comment]
@GBKS thank you, man! It's such a pleasure to hear that you already came up with this years ago! By the way: Your idea for the `transactions` page (option 1) looks INCREDIBLY beautiful!

> I don't have a super strong opinion on whether the language picker is a dropdown or an independent screen.

Me neither. Here are the 3 reasons why I did it as shown:
- A user who doesn't speak English can select his language without knowing the word "language". (Assuming he understands the sign ">" in "Next >")
- The current orange "Language" button directly next to the "Continue" button are too close. What the user can do on every wizard page should be in the middle - not hidden in the navigation area.
- I looked at other wizards and most of them would have a dedicated welcome screen, which puts the user in the center (not the software), I guess.

**Comment by selsta** [comment]
> The mode selection window forces you to select something but you can't go back.

#2520
